kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiSQL.g4

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
can we separate Lexer and Parser?

Reply:
Thanks, will separate Lexer and Parser files